### PR TITLE
Re-add support for AUTH_USER_MODEL environment variable

### PR DESCRIPTION
**Pull request by yakky**
I think support for custom user model has been removed from new manage.py

### CODE REVIEW

**Comment by ojii** [comment]
I think ideally this would be done with the built-in configuration system for django-app-manage, as for example is done for ``USE_TZ``: https://github.com/divio/django-cms/blob/develop/manage.py#L189-L193

See https://github.com/ojii/django-app-manage/blob/master/app_manage/config.py#L75

This way it will automatically support either env vars or a CLI argument.

Probably could look something like:

```
Config(
    env='AUTH_USER_MODEL',
    arg='--auth-user-model',
    default='django.contrib.auth.models.User'
)

**Comment by yakky** [comment]
@ojii should I send a PR to django-app-manage?

**Comment by yakky** [comment]
@ojii how can we add the different applications in this case?
Do we need a second setting?

**Comment by ojii** [comment]
will think of an elegant way to solve this

File: manage.py [hunk 1]
```diff
@@ -169,6 +178,11 @@
             'mti_pluginapp': 'cms.test_utils.project.mti_pluginapp.migrations',
         }
 
+    if os.environ.get('AUTH_USER_MODEL', False):
+        custom_user_app = 'cms.test_utils.project.' + os.environ.get('AUTH_USER_MODEL', False).split('.')[0]
+        INSTALLED_APPS.insert(INSTALLED_APPS.index('cms'), custom_user_app)
```
Comment (ojii) [inline manage.py]: is this order important? (insert over append)

Comment (yakky) [inline manage.py]: Yes, it is: http://django-cms.readthedocs.org/en/develop/reference/configuration.html#custom-user-requirements

**Comment by ojii** [comment]
See #4370 for my implementation of this.

**Comment by yakky** [comment]
Fixed by #4370

**Pull request closed without merging**